Tests/AppTests/SearchTests.swift

```diff
@@ -156,7 +156,7 @@ class SearchTests: AppTestCase {
     func test_query_sql() throws {
         // Test to confirm shape of rendered search SQL
         // MUT
-        let query = Search.query(app.db, ["test"], page: 1, pageSize: 20)
+        let query = try XCTUnwrap(Search.query(app.db, ["test"], page: 1, pageSize: 20))
```

Comment:
Not a huge change really, just making clearer what the failure is in case the query is `nil`.